src/index.html

```diff
@@ -7,8 +7,58 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous"
+    >
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#logo" class="logo-link">
+        <img
+          class="logo-image"
+          src="images/logo.png"
+          alt="Logo_moyo-header"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav_list">
+          <li class="nav_item">
+            <a href="#apple" class="nav__link_is-active">Apple</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#samsung" class="nav__link">Samsung</a>
+          </li>
+
+          <li class="nav_item">
+            <a href="#smartphones" class="nav__link">Smartphones</a>
+          </li>
+
+          <li class="nav_item" data-qa="hover">
+            <a href="#laptops-and-computers" class="nav__link">Laptops & computers</a>
```

Comment:
Attribute should be with `a` tag 
![image](https://github.com/mate-academy/layout_moyo-header/assets/70009205/49a19456-8e20-4f3e-a7c3-294ca9226160)
```suggestion
            <a 
              href="#laptops-and-computers" 
              class="nav__link"
              data-qa="hover"
            >
              Laptops & computers
            </a>
```

Reply:
Done